pkg/analyzer/filter/eval.go

```diff
@@ -55,8 +60,37 @@ func NewEvaluator(name string, ignoreError bool) (Evaluator, error) {
 		cel.Variable(filterInputVarScorecard, cel.DynType),
 		cel.Variable(filterInputVarLicenses, cel.DynType),
 		cel.Variable(filterInputVarRoot, cel.DynType),
+		cel.Function("contains_license",
+			cel.MemberOverload("list_string_contains_license_string", []*cel.Type{cel.ListType(cel.StringType), cel.StringType}, cel.BoolType,
+				cel.BinaryBinding(func(lhs, rhs ref.Val) ref.Val {
+					l := lhs.(traits.Lister)
+					filterlicenseexp := fmt.Sprintf("%s", rhs)
+					iter := l.Iterator()
+					contains := false
+					i := 0
+					for {
+						if contains {
+							break
+						}
+						if iter.HasNext().Value() == true {
+							str := l.Get(types.Int(i))
+
+							licenseexp := fmt.Sprintf("%s", str)
+							extracted, _ := spdxexp.ExtractLicenses(licenseexp)
```

Comment:
Do not skip error handling using `_`. Handle error by failing. I think `return fmt.Errorf` is appropriate here but you should read: https://go.dev/blog/error-handling-and-go

Reply:
I have handled it by fatalf from logger as that function only expects boolean return value